Cargo.toml

```diff
@@ -23,10 +23,12 @@ byteorder = "1.3.2"
 failure = "0.1.6"
 hex_fmt = "0.3.0"
 log = "0.4.8"
-pairing = { version = "0.14.2", features = ["u128-support"] }
-rand = "0.6.5"
-rand04_compat = "0.1.1"
-rand_chacha = "0.1.1"
+pairing = { version = "0.15.0" }
```

Comment:
Let's keep it in alphabetical order.
Also, whenever only a version is specified, let's use the simple form:
```toml
pairing = "0.15.0"
```